man/memtier.7

```diff
@@ -3,14 +3,19 @@
 .\"
 .TH "MEMTIER" 7 "2021-03-01" "Intel Corporation" "MEMTIER" \" -*- nroff -*-
 .SH "NAME"
-libmemtier.so \- TODO
+libmemtier.so - internal library which enables memory tiering
 
 .SH "SYNOPSIS"
-.BR LD_PRELOAD=libmemtier.so
+In order to enable the usage of tiering in memkind, pass the following environment variables along with the command:
```

Comment:
If "memory tiering" is established, then it should be "the usage of memory tiering in memkind"

Reply:
Done.